includes/views/listing_contact.php

```diff
@@ -53,6 +53,10 @@ private function validate() {
             $this->errors[] = _x( 'Please enter your name.', 'contact-message', 'business-directory-plugin' );
         }
 
+        if ( ! $this->email ) {
+            $this->errors[] = _x( 'Please enter your email.', 'contact-message', 'business-directory-plugin' );
```

Comment:
Nice job! Maybe we can combine this so there is one less string to translate?

`if ( ! $this->email || ! wpbdp_validate_value( $this->email, 'email' ) ) {`